build.gradle

```diff
@@ -1,10 +1,12 @@
 buildscript {
-    repositories {
+  ext.kotlin_version = '1.1.4-2'
+  repositories {
```

Comment:
AFAIK it uses 4 spaces indentation while library uses 4 and sample uses 2. The code is indented to 2.
I didn't want to change too much lines but I think is necessary a little reformat on those two gradle files. What do you think?